docs/api.md

```diff
@@ -1895,6 +1895,12 @@ This is a shortcut for [page.mainFrame().url()](#frameurl)
 #### page.waitFor(selectorOrFunctionOrTimeout[, options[, ...args]])
 - `selectorOrFunctionOrTimeout` <[string]|[number]|[function]> A [selector], predicate or timeout to wait for
 - `options` <[Object]> Optional waiting parameters
+  - `visible` <[boolean]> wait for element to be present in DOM and to be visible, i.e. to not have `display: none` or `visibility: hidden` CSS properties. Defaults to `false`.
```

Comment:
What about the `hidden` attribute?

Maybe it's better to not explicitly list the examples (like `display: none` or `visibility: hidden`) here.

Reply:
Changed as requested. Examples are not listed anymore. Just shows default value and description.